README.md

```diff
@@ -147,6 +153,7 @@ The cmdlet does not fully support the Inquire action for debug messages. Cmdlet
 
 * Added the following resources:
     * MSFT_xNetworkTeam resource to manage native network adapter teaming.
+    * MSFT_xHostsFile resource to manage hosts file entries.
```

Comment:
This will need to be moved to Unreleased as 2.7.0.0 has been released to the gallery.